src/Kernel.package/AsciiCharset.class/class/isLetter..st

```diff
@@ -1,4 +1,4 @@
-casing
+character classification
```

Comment:
I don't see how this version or the original could be correct. a-z and A-Z should both be included.

Reply:
It sure looks wrong to me too.  I think that this was an existing bug in Pharo 7.